### PR TITLE
#347 homogeneize doxygen generation with others project and fails at

### LINKED ISSUE #347: [Compilation] Stop compilation with an error when there are doxygen warnings

### PR DESCRIPTION
**Pull request by rosiereflo**
compilation if some warnings are found
closes #347

### CODE REVIEW

File: doxygen/footer.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!--
+    Copyright (c) 2022, RTE (http://www.rte-france.com)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source time domain
```
Comment (marcochiaramello) [inline doxygen/footer.html:10]: same as above

File: doxygen/mainPage.dox [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
```
Comment (marcochiaramello) [inline doxygen/mainPage.dox:10]: same as above

File: doxygen/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2022, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
```
[the remark this answers is not in the document]
Comment (rosiereflo) [reply doxygen/CMakeLists.txt]: Solved

File: doxygen/footer.html (hunk 1 above)
Comment (rosiereflo) [reply doxygen/footer.html]: Solved

File: doxygen/mainPage.dox (hunk 2 above)
Comment (rosiereflo) [reply doxygen/mainPage.dox]: Solved

**Review by marcochiaramello — APPROVED** [review]

**Pull request merged**